app/src/main/java/org/stepik/android/presentation/course_reviews/CourseReviewsPresenter.kt

```diff
@@ -83,7 +83,12 @@ constructor(
             .subscribeOn(backgroundScheduler)
             .observeOn(mainScheduler)
             .subscribeBy(
-                onSuccess = { state = it },
+                onSuccess = {
```

Comment:
we should request new course reviews only if we open that screen (set user visible hint )